supertokens_python/recipe/jwt/recipe_implementation.py

```diff
@@ -52,7 +52,7 @@ async def create_jwt(
         data = {
             "payload": payload,
             "validity": validity_seconds,
-            "use_static_signing_key": use_static_signing_key is not False,
+            "useStaticSigningKey": use_static_signing_key is not False,
```

Comment:
We should add a test for this

Reply:
> We should add a test for this

Have test for `create_jwt(payload, jwt_expiry, use_static_signing_key=False)` can add for `create_jwt(payload, jwt_expiry, use_static_signing_key=True)` (it's the default anyways)

TODO

Reply:
No I meant to add a test that checks that a static signing key is/is not used, because using `use_static_signing_key` would not have made a difference since the core would have ignored it

Reply:
Done.